gnucash/gnome/dialog-invoice.c

```diff
@@ -1949,6 +1948,11 @@ gnc_invoice_update_window (InvoiceWindow *iw, GtkWidget *widget)
         gtk_widget_set_sensitive (iw->notes_text, TRUE);
     }
 
+    if(gncInvoiceIsPaid (invoice))
+            gtk_label_set_text(GTK_LABEL(iw->paid_label),  _("PAID"));
```

Comment:
Indentation.

Reply:
Thanks John. Will do.

Reply:
It would save the some translation if we would use some `uppercase (_("paid"|"unpaid))` function in this module.
Also a translator comment  about the context is always welcome for single words.